resources/sass/components/_selects.scss

```diff
@@ -8,12 +8,22 @@
     position: relative;
     vertical-align: middle;
 
+    @include breakpoint($bp-mobile-max) {
+        margin-left: $indent-s;
+        min-width: 42vw;
+    }
+
+    @include breakpoint($bp-tablet-min) {
+        margin-left: $indent-sm;
+        min-width: 240px;
+    }
+
     &::before {
-        color: $c-gray2;
-        content: 'L';
+        color: $c-secondary;
+        content: 'e';
```

Comment:
better to use separate element for icons, for example <span class="..."></span> and not redefine icon on the whole component with :after or :before